### ISSUE TITLE
Automated CKAN Job Error Condition

### ISSUE THREAD

**Issue by nickumia-reisys** [GSA/catalog.data.gov#967]
Workflow with Issue: 4 - Automated CKAN Jobs
Job Failed: ckan-auto-command
CKAN Command (in question): ckan harvester run
CKAN Command Schedule: 4/15 * * * *
Cloud.gov Environment: prod

Last Commit: b7644b9b5f5b0d4e354e66d0789308aaf4c8775b
Number of times run: 1
Last run by: jbrown-xentity
Github Action Run: https://github.com/GSA/catalog.data.gov/actions/runs/5329411648

**Comment by robert-bryson** [comment]
> Any reason we shouldn't turn off dashboard (at least in staging and dev) @GSA/data-gov-team ?

No, other than it is in the [deprecate ticket](https://github.com/GSA/data.gov/issues/3502) that hasn't been taken on yet.
